src/main/java/org/jenkinsci/extension_indexer/ExtensionPointListGenerator.java

```diff
@@ -56,11 +49,8 @@
      */
     private final Map<MavenArtifact,Module> modules = Collections.synchronizedMap(new HashMap<MavenArtifact,Module>());
 
-    @Option(name="-wiki",usage="Generate the extension list index and write it out to the specified file.")
```

Comment:
So you drop the Wiki mode at all? Fine for me though I would rather keep it since the code was already there